scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -69,6 +69,9 @@ namespace <%= namespace %> {
 
         private IWebCheckout WebCheckout;
         private INativeCheckout NativeCheckout;
+        private PaymentSettings StorePaymentSettings;
+
+        private ShopifyClient CurrentClient;
```

Comment:
Nit: No need to call this `Current` since we're note re-assigning it or anything in the future.